.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature Request
+about: Create a feature request
+title: Feature Proposal
+labels: enhancement
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the feature request or improvement in the Title above -->
+<!--- Look through existing open and closed feature proposals to see if someone has asked for the feature before -->
+
+## Expected Behavior
+<!--- Tell us how it should work -->
+
+## Difference from Current Behavior
```

Comment:
if it is feature, normally the request just doesn't exist today? not sure if we need a difference section as I assume it is applicable to bug?

Reply:
True, removing this one